### PR TITLE
Enable DeviceLab arm64 ios tests as `bringup: true`

### PR DESCRIPTION
**Pull request by keyonghan**
This PR enables passing arm64 tests (staging) in prod. Initially as `bringup: true`, they will be marked as blocking when validated.

Context:
https://flutter-review.googlesource.com/c/infra/+/26927 enabled m1 tests in staging.
https://chrome-internal-review.googlesource.com/c/infradata/config/+/4680019 brought up 4 m1/ios testbeds in prod.
These tests have been passing consistently in staging:
https://ci.chromium.org/p/flutter/builders/staging/Mac_arm64_staging%20hot_mode_dev_cycle_macos_target__benchmark
https://ci.chromium.org/p/flutter/builders/staging/Mac_arm64_staging%20flutter_gallery_ios__compile
https://ci.chromium.org/p/flutter/builders/staging/Mac_arm64_staging%20hello_world_ios__compile
https://ci.chromium.org/p/flutter/builders/staging/Mac_arm64_staging%20native_ui_tests_ios
https://ci.chromium.org/p/flutter/builders/staging/Mac_arm64_staging%20build_ios_framework_module_test
https://ci.chromium.org/p/flutter/builders/staging/Mac_arm64_staging%20run_release_test_macos
https://ci.chromium.org/p/flutter/builders/staging/Mac_arm64_staging%20ios_app_with_extensions_test
https://ci.chromium.org/p/flutter/builders/staging/Mac_arm64_staging%20ios_content_validation_test
https://ci.chromium.org/p/flutter/builders/staging/Mac_arm64_staging%20macos_chrome_dev_mode

Issue: https://github.com/flutter/flutter/issues/87508

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -2478,6 +2501,19 @@ targets:
...
+    recipe: devicelab/devicelab_drone
+    presubmit: false
+    bringup: true # https://github.com/flutter/flutter/issues/87508
+    timeout: 60
+    properties:
+      tags: >
+        ["devicelab","ios","mac","arm64"]
+      task_name: build_ios_framework_module_test
+    runIf:
+      - dev/**
```
Comment (jmagman) [inline .ci.yaml:2514]: `runIf` should match their `x86` equivalent 
```suggestion
      - dev/**
      - packages/flutter_tools/**
      - bin/**
      - .ci.yaml
```

Comment (keyonghan) [reply .ci.yaml:2514]: Updated all builders to be consistent with x86.

**Comment by keyonghan** [comment]
A friendly ping.

**Review by jmagman — APPROVED** [review]
LGTM, thanks @keyonghan!

**Pull request merged**